docs/vc-http-api.yml

```diff
@@ -113,6 +113,65 @@ paths:
           description: invalid input!
         "500":
           description: error!
+  /presentations/available:
+    post:
+      tags:
+        - Holder
+      summary: Notifies a holder of an available presentation.
```

Comment:
Can we replace "holder" with "potential holder" since the notification can be ignored and language matters. @OR13

Reply:
can you make the proposed change on the PR and I will accept?